mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go.tmpl

```diff
@@ -992,6 +1009,25 @@ func ResourceBigQueryTable() *schema.Resource {
 				DiffSuppressFunc: bigQueryTableSchemaDiffSuppress,
 				Description:      `A JSON schema for the table.`,
 			},
+			{{- if ne $.TargetVersionName "ga" }}
+			// SchemaForeignTypeInfo: [Optional] Specifies metadata of the foreign data type definition in field schema.
+			"schema_foreign_type_info": {
```

Comment:
Does this field support updates? If so, could we add an update step in the test?

Reply:
It doesn't and API will throw an error. The behavior might change between Public Preview and GA or post-GA, so we are also not making it force-new right now.

Reply:
Thanks for confirming. I suggest we set it to `forcenew` for now and remove it once the update is supported. Otherwise users won't be able to change this field even if they want to delete and recreate the resource to align with the desired state. However, if the concern is to avoid accidentally deleting the table and the update will be supported soon, we could keep it as is for now